cluster-deploy/example-dir-tree/meta-service/conf/jdbc.properties

```diff
@@ -0,0 +1,21 @@
+#
```

Comment:
This file can be deleted. All jdbc connection properties can be move to ${module}.properties. In this case, the meta-service.properties.